.github/scripts/rust/Makefile.toml

```
@@ -179,6 +180,10 @@ private = true
 condition = { channels = ["nightly"] }
 install_crate = { rustup_component_name = "miri" }
 
-[tasks.install-nextest]
+[tasks.install-hack]
```

should be 
```suggestion
[tasks.install-cargo-hack]
```
according to the dependency, right?

---

Just noticed, yes. Thank you!